Gemfile.lock

```diff
@@ -339,3 +339,9 @@ DEPENDENCIES
   uglifier (>= 1.0.3)
   will_paginate (>= 3.0.6)
   will_paginate-bootstrap (>= 1.0.1)
+
+RUBY VERSION
```

Comment:
I'd like to not include Gemfile or Gemfile.lock changes if not essential, but I'll leave these in this time. Thanks!